models/profile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = (sequelize, DataTypes) => {
+  const Profile = sequelize.define('Profile', {
+
+    firstName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+    lastName: {
+      type: DataTypes.STRING,
+      allowNull: false
+    },
+
+    imageUrl: {
+      type: DataTypes.STRING,
+      allowNull: false,
+      validate: {
+        notEmpty: {
+          args: true,
+          msg: 'Image cannot be null, enter true or false!'
+        },
+      }
+    },
+
+    bio: {
+      type: DataTypes.STRING,
+      allowNull: false
```

Comment:
Is the bio enforced as well? I ask because if a user is created afresh, there isn't the option of creatign a bio at the point of user creation, and so there should be allowance for the bio being initially false until updated.